py/client-ticking/setup.py

```diff
@@ -59,9 +59,11 @@ def _compute_version():
     ext_modules = cythonize(
         [Extension("pydeephaven_ticking._core",
                    sources=["src/pydeephaven_ticking/*.pyx"],
+                   language="c++",
                    extra_compile_args=["-std=c++17"],
                    libraries=["dhcore_static"]
         )]),
     python_requires='>=3.8',
-    install_requires=[f"pydeephaven=={_version}"]
+    install_requires=[f"pydeephaven=={_version}"],
+    package_data={'pydeephaven_ticking': ['py.typed']}
```

Comment:
Afaict, this didn't work. I don't see a py.typed file in the output wheels.

Reply:
Apparently one needs an empty `py.typed` file in the source directory for the module also; `py/client` has that, so I did the same.